docs/UserGuide.md

```diff
@@ -143,6 +143,9 @@ Adds a client to the FitBook.
 
 Format: `add n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [w/WEIGHT] [h/HEIGHT] [nt/NOTE] [t/TAG]…​`
 
+
+* HEIGHT and WEIGHT each takes in only one value. (e.g. `add n/Tom p/123 w/85 65` or `add n/Tom p/123 h/180 175` is invalid).
```

Comment:
```suggestion
* HEIGHT and WEIGHT only take in one value each. (e.g. `add n/Tom p/123 w/85 65` or `add n/Tom p/123 h/180 175` is invalid).
```

Reply:
If its not too much, separating them onto their own lines will be better.